test/e2e/storage/sanity.go

```diff
@@ -555,8 +555,9 @@ var _ = Describe("sanity", func() {
 				v.publish(volName, vol)
 
 				// ensure the data retained
-				ssh = exec.Command(fmt.Sprintf("%s/_work/%s/ssh.%s", os.Getenv("REPO_ROOT"), os.Getenv("CLUSTER"), i))
-				ssh.Stdin = bytes.NewBufferString(`sudo sh -c 'cat ` + v.getTargetPath() + "/target/test-file" + `'`)
+				sshcmd := fmt.Sprintf("%s/_work/%s/ssh.%s", os.Getenv("REPO_ROOT"), os.Getenv("CLUSTER"), i)
+				cmd := "sudo cat " + v.getTargetPath() + "/target/test-file"
```

Comment:
This looks simpler and I can't think of a reason why this command would have to be passed via stdin. It's sometimes necessary when executing complex scripts where special shell characters just don't make it through the ssh command line, but not here.

@avalluri, you wrote this - was there a reason for doing it that way?